src/ripple/protocol/AnyPublicKey.h

```diff
@@ -102,9 +102,16 @@ class AnyPublicKey
 
 public:
     AnyPublicKey() = delete;
-    AnyPublicKey (AnyPublicKey const&) = delete;
     AnyPublicKey& operator= (AnyPublicKey const&) = delete;
 
+    AnyPublicKey (AnyPublicKey const& other)
+        : buffer_type(other.buffer_type::member.data(),
+            other.buffer_type::member.size())
+        , AnyPublicKeySlice (buffer_type::member.data(),
+            buffer_type::member.size())
+    {
+    }
```

Comment:
(a) Copy construction is needed, but copy assignment isn't.  (b) I tried to implement copy assignment and failed — I'm not sure `Buffer` allows for it.  I'd prefer to leave something more substantive than my uncertainty in a code comment… ideas?